lms/djangoapps/instructor/utils.py

```diff
@@ -185,16 +183,32 @@ def collect_ora2_data(course_id):
 
 
 # pylint: disable=invalid-name
-def ora2_data_queries():
+def ora2_data_queries(is_anonymous):
     """
     Wraps a raw SQL query which retrieves all ORA2 responses for a course.
     """
-
-    RAW_QUERY = """
```

Comment:
I would really prefer if we could maintain the existing structure of a single, parameterized SQL query here.  To my knowledge, this is the only section of the codebase in edx-platform which makes use of raw SQL querying, which I think is kind of dangerous, both from a SQL injection and possibility of people modifying it in the future without paying close attention.  At the very least, I think we can wrap RAW_QUERY_1 and RAW_QUERY_2 into a single string parameterized by RAW_QUERY_3 and course_id, but if possible, I'd like to keep all of the SQL in a single variable.